docs/adding_benchmarks_guide.md

```diff
@@ -291,6 +295,60 @@ address that the SinkBinding can use.
 - While deploying a Trigger, best wait until both the _broker_ and the _subscriber_ ("receiver") 
 are ready.
 
+## Benchmarking
```

Comment:
Let's move benchmarking to a separate doc, leaving a reference to it in this doc.

We need an overview subsection that describes the methodology (you can show your diagram with prod, cons and timeseriesDB here):
- the invoker steers load to a collection of serverless app
- apps can comprise a single function, a collection of functions calling one another, as well as asynchronous workflows and pipelines
- for synchronous single- and multi-function apps, the invoker measures the latency e2e
- for async, the invoker registers a workflow with a timeseriesDB service. for each invocation, the invoker takes a timestamp of the workflow being invoked whereas TimeseriesDB waits for the corresponding completion events (give simple examples that are supported) and takes a timestamp when the last completion event is received.
- The invoker then retrieves all the records of all async workflows, and adds these records to the same trace as for the sync apps.
- then goes the guide as below